app/(auth)/logout/logoutAction.ts

```diff
@@ -0,0 +1,18 @@
+'use server';
+import { cookies } from 'next/headers';
+import { deleteSessionByToken } from '../../../database/sessions';
+
+
+export default async function setCookie() {
+  const cookieStore = cookies();
+  const token = cookieStore.get('sessionToken');
+
+  if (token) {
+    await deleteSessionByToken(token.value);
+  }
+
+  (cookieStore as any).set('sessionToken', '', {
+    maxAge: -1,
+    expires: new Date(Date.now() - 10000),
```

Comment:
why `expires` with this unusual value?

seems like we've done it more simply in the past:

https://github.com/upleveled/next-js-example-fall-2022-vienna-austria/blob/b7b24517b108d16dc311b42a72c015b60da64d13/pages/logout.tsx#L17-L23